ui/src/app.js

```diff
@@ -1767,10 +1773,10 @@ function shouldEnableTouch () {
   return chromiumPlus73 || chromePlus73
 }
 
-function setAvailableCoupons (areThereAvailableCoupons) {
-  if (areThereAvailableCoupons) {
-    $('#insert-first-bill-coupon-added').hide()
-    $('#choose-fiat-coupon-added').hide()
+function setAvailablePromoCodes (areThereAvailablePromoCodes) {
```

Comment:
why not pull the if inside the else { **if {}** } outside?

like if (a) {} else if (b) else {}